test/e2e/Rakefile

```diff
@@ -231,33 +231,46 @@ task :display_versions do
   end
 end
 
-task :get_latest_bins, [:pr] do |_task, args|
-  log ">> Getting latest node and wallet binaries from Hydra into #{BINS}"
-
-  pr = args[:pr] || 'master'
-  if pr.empty? || pr == 'master'
-    log 'From master branch 👇'
-    pr_num = nil
-  else
-    log "From PR: ##{pr} 👇"
-    pr_num = pr.to_i
-  end
-  wget(get_latest_binary_url(pr_num))
-
-  if win?
-    mk_dir(BINS)
-    cmd "unzip -o binary-dist -d #{BINS}"
-    cmd "#{BINS}/cardano-node.exe version"
-    cmd "#{BINS}/cardano-wallet.exe version"
-  else
-    Rake::Task[:clean_bins].invoke
-    mk_dir(BINS)
+task :get_latest_bins, [:branch] do |_task, args|
+  log ">> Getting latest node and wallet binaries from Buildkite into #{BINS}"
+  branch = args[:branch] || 'master'
+  bk = Buildkite.new
+  last_build = bk.last_build_number(branch: branch, state: 'passed')
+  log "Latest passing buildkite '#{bk.pipeline}' pipeline for branch '#{branch}' is #{last_build}"
+  jobs = bk.jobs(last_build)
+  mk_dir(BINS)
+
+  if linux?
+    job_name = 'Build cardano-wallet package for Linux'
```

Comment:
JFYI: buildkite jobs have human-readable titles as well as machine-oriented `key` values. In order to make this integration less fragile (like what if somebody changes the job title?) I'd suggest to obtain job by its key instead.